05_stringer/CameronSchafer/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main()
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "127.0.0.1\n"
+	actual := buf.String()
+
+	assert.Equalf(t, expected, actual,
+		"Unexpected output in main()\nexpected: %q\nactual: %q",
+		expected, actual)
+}
+
+// String()
+func TestFmtStringer(t *testing.T) {
+	testCases := map[string]struct {
+		input       IPAddr
+		expectedArr string
+	}{
+		"127.0.0.1": {input: IPAddr{127, 0, 0, 1},
```

Comment:
You could test boundary cases like single value, no value, etc.

Reply:
Done,
Thank you for the hint.